app/views/teachers/mailer/otp.text.erb

```diff
@@ -1,4 +1,6 @@
-Use this code to confirm your email address:
+<b> This is an automated email, please do not reply. </b>
```

Comment:
We can't use HTML tags in email formatting as Notify doesn't support them - is there an alternative formatting we can use?

https://www.notifications.service.gov.uk/using-notify/guidance/edit-and-format-messages#formatting

Reply:
I'll check with dom. Perhaps it can be a heading in stead of bold